### ISSUE TITLE
Project No Longer Maintained?

### ISSUE THREAD

**Issue by cardfaux** [Shopify/shopify-express#129]
I see this project is no longer maintained and it says to use packages with Koa, Does shopify no longer support making apps with express?

**Comment by paul-phan** [comment]
Yes, it will not be maintained anymore. I already do another maintenance for this and using it in our product, you can check out this repo: https://github.com/paul-phan/shopify-express-x
